### ISSUE TITLE
Can't delete an archived project

### ISSUE THREAD

**Issue by loelkes** [kromitgmbh/titra#69]
If I delete a project I get this error in the Console and the project remains:

```
{
    "file": "/imports/ui/pages/projectlist.js",
    "onLine": 1,
    "browserInfo": {
        "browser": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36",
        "userId": "BXJPeERSRfxGyYs8X",
        "url": "http://localhost:4000/list/projects",
        "resolution": "1792x1120"
    }
}
```
And then this
```
09:41:09.161 /imports/ui/pages/projectlist.js:1 Uncaught TypeError: Cannot read property 'fire' of undefined
    at Object.click .js-delete-project (/imports/ui/pages/projectlist.js:1)
    at 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156
    at Function.c._withTemplateInstanceFunc (6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156)
    at d.View.<anonymous> (6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156)
    at 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156
    at Object.d._withCurrentView (6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156)
    at d._DOMRange.<anonymous> (6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156)
    at HTMLButtonElement.delegatedHandler (6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156)
    at HTMLDivElement.dispatch (6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:34)
    at HTMLDivElement.g.handle (6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:34)
click .js-delete-project @ /imports/ui/pages/projectlist.js:1
(anonymous) @ 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156
c._withTemplateInstanceFunc @ 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156
(anonymous) @ 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156
(anonymous) @ 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156
d._withCurrentView @ 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156
(anonymous) @ 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156
delegatedHandler @ 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:156
dispatch @ 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:34
g.handle @ 6078148c3b09b77d3c61cf16dd8f9eddea1d1cac.js?meteor_js_resource=true:34
```

My setup:

```
titra: 0.52.0
Node.js: 12.22.1
Meteor.js: 2.2
MongoDB: 4.2.15
```

**Comment by faburem** [comment]
Thanks for reporting this issue. It will be fixed in titra 0.53.0!

**Comment by faburem** [comment]
Fixed in titra 0.53.0 (currently rolling out on docker hub)

**Comment by loelkes** [comment]
LOL, that was fast. Awesome! Thank you.
